catalog-info.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: cloudcost-exporter
+  title: Cloudcost Exporter
+  annotations:
+    backstage.io/techdocs-ref: dir:./docs
```

Comment:
I think this should be `backstage.io/techdocs-ref: dir:.`, with a `mkdocs.yml` file at the root to configure the docs generation (like https://github.com/grafana/cog/blob/main/mkdocs.yml)

Also: I think there should be an index.md file in every folder under `docs/` for the docs to be generated properly (or just for the navigation?)

You should be able to preview the docs locally with `npx @techdocs/cli serve -v`